news/printing-info.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* Added test for ``print_info`` function.
```

Comment:
This can just say "added ``print_info`` function. "    it is understood that it will have a test.